libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -3,6 +3,7 @@
 <ng-template #desktopTemplate>
     <fd-popover
         additionalBodyClass="fd-popover-custom-list"
+        [class.fd-combobox-full-width]="isSearch"
```

Comment:
Shouldn't this be up to the developer? If we are making this decision for them, not sure why it'd only apply to search comboboxes and not all of them

Reply:
That's a good question. I also thought about it during the implementation. Why I did it this way? Combobox has restrictions when it comes to max and min width, which do not apply on Search Field. Multi Input, for example, also takes the 100% of the container. Making Search Field "full-width" we give the option to the user to apply whatever width they want on the container inside which they gonna place the Search field and this way also remove the css rules that apply on the Compobox when in comes to the width. Does this make sense?

Reply:
Yes! PR looks good